### PR TITLE
Add freeipa roles for PvC pre_setup RHEL only

### PR DESCRIPTION
**Pull request by clevesque**
Add freeipa roles for PvC pre_setup RHEL only

Signed-off-by: Chuck Levesque <clevesque@cloudera.com>

### CODE REVIEW

File: roles/freeipa_client/meta/main.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+galaxy_info:
+  role_name: freeipa_client
+  namespace: cloudera
+  author: Webster Mudge (wmudge@cloudera.com)
```
Comment (wmudge) [inline roles/freeipa_client/meta/main.yml:20]: Update `authors`

File: roles/freeipa_client/meta/main.yml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  description: >
```
Comment (wmudge) [inline roles/freeipa_client/meta/main.yml:21]: Update `description`

File: roles/freeipa_client/meta/main.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  - name: Fedora
+    versions: all
+  - name: GenericLinux
+    versions: all
+  - name: MacOSX
+    versions: all
+  - name: Ubuntu
+    versions: all
+
+  galaxy_tags:
```
Comment (wmudge) [inline roles/freeipa_client/meta/main.yml:44]: Update `tags`

File: roles/freeipa_server/meta/main.yml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+galaxy_info:
+  role_name: freeipa_server
+  namespace: cloudera
+  author: Webster Mudge (wmudge@cloudera.com)
+  description: >
```
Comment (wmudge) [inline roles/freeipa_server/meta/main.yml:21]: Update `description`

File: roles/freeipa_server/meta/main.yml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  - name: Fedora
+    versions: all
+  - name: GenericLinux
+    versions: all
+  - name: MacOSX
+    versions: all
+  - name: Ubuntu
+    versions: all
+
+  galaxy_tags:
```
Comment (wmudge) [inline roles/freeipa_server/meta/main.yml:44]: Update `tags`

**Review by wmudge — CHANGES_REQUESTED** [review]

File: roles/freeipa_client/meta/main.yml (hunk 1 above)
Comment (clevesque) [reply roles/freeipa_client/meta/main.yml:20]: the author really is JIm, I will put him

File: roles/freeipa_client/meta/main.yml (hunk 3 above)
Comment (clevesque) [reply roles/freeipa_client/meta/main.yml:44]: what should the tags be?

**Comment by jimright** [comment]
FYI @clevesque and @wmudge - I've tested these roles to deploy PvC on AWS with sidecar FreeIPA and all works well. Once the suggested changes are made I see no issue in approving and merging.

**Comment by clevesque** [comment]
Done

**Review by wmudge — APPROVED** [review]

**Pull request merged**